### ISSUE TITLE
After Back button, registration UI stuck at dashboard page

### ISSUE THREAD

**Issue by linggao** [open-horizon/anax-ui#34]
I put correct info at the first page then clicked "Register Device with Existing Account" ,  it moved to the second page. I then click the back button on the browser, it went to the dashboard. I can no longer get to the second page.

**Comment by bmpotter** [comment]
Varatep, the behavior ling describes above is the same behavior i saw earlier. I just tried it again with snap 73, and i went all the way to the 3rd page (poc list), clicked the browser back button and successfully got back to the 2nd page (register device settings). Then I clicked the browser back button again and instead of taking me to the 1st page, it brought me to a page that said "Your device is not configured to execute any services" and had 1 button labelled "Begin Setup". Clicking this took me to the 2nd page, and i could continue on from there. (This was all on firefox.)

So at no point did i get stuck on the dashboard like before. I guess the only issue is that i couldn't get back to the 1st page if i changed my mind about what username i wanted to use. But i can see that's more involved because the device object is already created and would probably need to be deleted before starting over. Also, since it is a rare case that someone successfully got thru the username creation or sign-in page successfully and want to go back, i think this issue can go on the less important list (to be done by mike when he's back, or by you when you are done with your poc ui work).
